Boardfiles/nucleol552zeq/Core/Src/main.cpp

```diff
@@ -125,7 +126,7 @@ int main(void)
   MX_TIM5_Init();
   MX_ADC1_Init();
   MX_ICACHE_Init();
-  MX_IWDG_Init();
+  //MX_IWDG_Init();

```

Comment:
Any reason we needed to comment this out?

Reply:
Yeah I can uncomment that, Hardy just told us to comment it out before we started debugging. I think he said it would cause errors when we try to run the code.